R/arc-add-update-delete.R

```diff
@@ -60,7 +60,7 @@ add_features <- function(
   # initial check for type of `x`
   obj_check_layer(x)
 
-  stopifnot("`.data` must be a data.frame-type class" = inherits(.data, "data.frame"))
+  check_dataframe(.data)
```

Comment:
This is the only location where check_dataframe() is used. As such should be removed as unnecessary code

Reply:
You don't think there are any other cases where you may use it in the future?

Reply:
There a good chance! But if I don't want to add additional code and more functions until its absolutely necessary. If it will be used in 2+ places then I'm game!